lib/generators/suspenders/install/web_generator.rb

```diff
@@ -0,0 +1,67 @@
+module Suspenders
+  module Generators
+    module Install
+      class WebGenerator < Rails::Generators::Base
+        include Suspenders::Generators::APIAppUnsupported
+        include Suspenders::Generators::DatabaseUnsupported
+
+        desc <<~MARKDOWN
+        Invokes all necessary generators for new Rails applications generated with Suspenders.
+
+        This generatator is intended to be invoked as part of an [application template][].
+
+        ```
+        rails new suspenders_qa \
+        --skip-test \
+        -d=postgresql \
+        -m=https://raw.githubusercontent.com/thoughtbot/suspenders/main/lib/install/web.rb
+        ```
+
+        [application template]: https://guides.rubyonrails.org/rails_application_templates.html
+        MARKDOWN
+
+        def invoke_generators
+          # This needs to go first, since it configures `.node-version`
+          generate "suspenders:prerequisites"
+
+          generate "suspenders:accessibility"
+          generate "suspenders:advisories"
+          generate "suspenders:email"
+          generate "suspenders:factories"
+          generate "suspenders:inline_svg"
+          generate "suspenders:lint"
```

Comment:
Need to figure this out:

```
❯ yarn run lint
yarn run v1.22.17
$ run-p lint:eslint lint:stylelint lint:prettier
$ eslint --max-warnings=0 --no-error-on-unmatched-pattern 'app/javascript/**/*.js'
$ prettier --check '**/*' --ignore-unknown
$ stylelint 'app/assets/stylesheets/**/*.css'
Checking formatting...
.github/workflows/ci.yaml
Oops! Something went wrong! :(

ESLint: 9.0.0

Error: Could not find config file.
    at locateConfigFileToUse (/Users/polito/Desktop/suspenders_qa_0/node_modules/eslint/lib/eslint/eslint.js:349:21)
    at async calculateConfigArray (/Users/polito/Desktop/suspenders_qa_0/node_modules/eslint/lib/eslint/eslint.js:384:49)
    at async ESLint.lintFiles (/Users/polito/Desktop/suspenders_qa_0/node_modules/eslint/lib/eslint/eslint.js:814:25)
    at async Object.execute (/Users/polito/Desktop/suspenders_qa_0/node_modules/eslint/lib/cli.js:461:23)
    at async main (/Users/polito/Desktop/suspenders_qa_0/node_modules/eslint/bin/eslint.js:165:22)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
app/assets/stylesheets/application.postcss.cssERROR: "lint:eslint" exited with 2.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Relates to https://github.com/eslint/eslint/issues/18287

Reply:
I think we can merge this commit without a solution for the above, since that can be addressed separately, and is not related to with commit.